.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           - {VERSION: "3.11", TOXENV: "docs", OPENSSL: {TYPE: "openssl", VERSION: "3.0.7"}}
           - {VERSION: "pypy-3.8", TOXENV: "pypy3-nocoverage"}
           - {VERSION: "pypy-3.9", TOXENV: "pypy3-nocoverage"}
+          - {VERSION: "3.11", TOXENV: "py311", OPENSSL: {TYPE: "openssl", VERSION: "1.1.1c"}}
```

Comment:
Should we be adding this job now so we can drop focal without coverage loss later? 😄